dig.go

```diff
@@ -215,16 +215,25 @@ type VisualizeOption interface {
 
 var _graphTmpl = template.Must(template.New("DotGraph").Parse(`digraph {
 	graph [compound=true];
+	{{range $g := .Groups}}
+		{{- printf "%q" .String}} [shape=diamond label=<{{.Type}}{{.Attributes}}>];
+		{{range .Results}}
+			{{- printf "%q" $g.String}} -> {{printf "%q" .String}};
```

Comment:
This is probably fine for now but in a future change, we should maybe fix the
repetitive printfs. We can probably add methods to the underlying types.
Something like,

```go
func (n *Node) QuotedString() string {
    return strconv.Quote(n.String())
}
```

(`strconv.Quote(n.String())` is equivalent to `fmt.Sprintf("%q", n.String())`.)

Then in the template, you can do `{{.QuotedString}}` and it'll call the method.

Or you can add a `quote` function to the `template.FuncMap` (see [Template.Funcs][1])
and do `{{quote .String}}`.

[1]: https://golang.org/pkg/text/template/#Template.Funcs